specificatie/openapi.yaml

```diff
@@ -908,17 +908,43 @@ components:
           items:
             $ref: '#/components/schemas/ZoekResultaatHal'
 
-    IntegerFilter:
+    BouwjaarFilter:
       type: object
-      description: >-
-        Geeft aan met welke waarden je kunt zoeken.
-        Met `min` zoek je naar waarden groter of gelijk aan de opgegeven waarde.
-        Met `max` zoek je naar waarden kleiner dan de opgegeven waarde.
+      description: |
+        "Zoek binnen een bereik van de opgegeven minimumwaarde tot maximumwaarde. 
+        De opgegeven minimumwaarde moet lager zijn dan de opgegeven maximumwaarde."
+      properties:
+        min:
+          type: integer
+          minimum: 0
+          maximum: 9999
+          default: 0
+          description: "Zoek naar waarden groter of gelijk aan de opgegeven waarde."
+        max:
+          type: integer
+          minimum: 0
```

Comment:
Strict zou minimum 1 moeten zijn omdat in de description is aangegeven dat de minimum waarde van BouwjaarFilter lager moet zijn dan de maximum waarde. Verder geldt voor de min property dat de maximum waarde 9998 moet zijn

Reply:
Dat heeft helemaal geen zin, omdat het dan nog steeds mogelijk is om voor min een grotere waarde dan voor max op te geven. 
In de feature staat nu een regel dat min niet groter mag zijn dan max. Dat betekent dat min wel gelijk mag zijn aan max en dat min dus 9999 mag zijn omdat dit ook de maximale waarde voor max is.